udata/templates/territories/territory.html

```diff
@@ -38,6 +40,11 @@
                 </strong>
             </p>
         </div>
+        <a href="mailto?" type="button" class="btn btn-warning btn-follow btn-block btn-sm"
```

Comment:
Mmh I don't think it's consistent with the wording in that case, here we want the user to notify us that he is interested in participating for his territory. And I thought it was what we discussed during the demo. Did I miss something?